### PR TITLE
Using ordered list reversed to instead dict reserved.

### PR DESCRIPTION
**Pull request by vin-huang**
* python3.6 doest not support dict reserved
 [SWDEV-471018]( https://ontrack-internal.amd.com/browse/SWDEV-471018)

### CODE REVIEW

**Comment by KKyang** [comment]
Please check if there's any backport wheels for 3.6

**Comment by vin-huang** [comment]
the gfx94x is passed on ubuntu22

File: tensilelite/Tensile/KernelWriter.py [hunk 1]
```diff
@@ -4761,7 +4761,7 @@ def findInstByName(module, names, count):
     if len(found_phs) == 0:
       return
 
-    for count in reversed(found_phs.keys()):
+    for count in reversed(sorted(list(found_phs.keys()))):
```
Comment (KKyang) [inline tensilelite/Tensile/KernelWriter.py]: if sys.version_info[:3] > (3,7)::
    # do something
else:
    # do something else

**Review by KKyang — APPROVED** [review]

**Review by Serge45 — APPROVED** [review]

**Comment by vin-huang** [comment]
gfx942, local test passed on both hipblaslt-test and tensilelite tests.

**Pull request merged**